torchaudio/_extension.py

```diff
@@ -85,6 +86,19 @@ def _init_extension():
         warnings.warn("torchaudio C++ extension is not available.")
         return
 
+    # On Windows Python-3.8+ has `os.add_dll_directory` call,
+    # which is called to configure dll search path.
+    # To find cuda related dlls we need to make sure the
+    # conda environment/bin path is configured Please take a look:
+    # https://stackoverflow.com/questions/59330863/cant-import-dll-module-in-python
+    if os.name == "nt" and sys.version_info >= (3, 8):
```

Comment:
Is >= necessary or == would be sufficient? I thought we do not have this issue for python 3.9+?